src/components/structures/TimelinePanel.js

```diff
@@ -251,10 +251,12 @@ var TimelinePanel = React.createClass({
     },
 
     onMessageListUnfillRequest: function(backwards, scrollToken) {
+        // If backwards, unpaginate from the back
```

Comment:
this could do with being a little clearer: "If backwards, unpaginate from the back (ie, the start of the timeline)" ?